target/ppc/insn32.decode

```diff
@@ -427,3 +431,10 @@ XXSPLTW         111100 ..... ---.. ..... 010100100 . .  @XX2
 ## VSX Vector Load Special Value Instruction
 
 LXVKQ           111100 ..... 11111 ..... 0101101000 .   @X_uim5
+
+## VSX Comparison Instructions
+
+XSMAXCDP	111100 ..... ..... ..... 10000000 ...   @XX3
+XSMINCDP	111100 ..... ..... ..... 10001000 ...   @XX3
+XSMAXJDP	111100 ..... ..... ..... 10010000 ...   @XX3
+XSMINJDP	111100 ..... ..... ..... 10011000 ...   @XX3
```

Comment:
tab -> spaces
```suggestion
XSMAXCDP        111100 ..... ..... ..... 10000000 ...   @XX3
XSMINCDP        111100 ..... ..... ..... 10001000 ...   @XX3
XSMAXJDP        111100 ..... ..... ..... 10010000 ...   @XX3
XSMINJDP        111100 ..... ..... ..... 10011000 ...   @XX3
```

Reply:
doh, fixed in https://github.com/PPC64/qemu/pull/51/commits/7b280c144de38c0ab2cc8d0b6bf4d6cf42647d8b
Nothing to see here :)